src/analyzer/analyzerebur128.cpp

```diff
@@ -69,7 +69,13 @@ void AnalyzerEbur128::storeResults(TrackPointer pTrack) {
         qWarning() << "AnalyzerEbur128::storeResults() failed with" << e;
         return;
     }
-    if (averageLufs == -HUGE_VAL || averageLufs == 0.0) {
+    if (averageLufs == -HUGE_VAL ||
+            averageLufs == HUGE_VAL ||
+            // This catches inf and -inf that slipped through in libebur for some reason..
+            // TODO Move this to a VERIFY_OR_DEBUG_ASSERT() once the root cause
+            // has been identified.
+            util_isinf(averageLufs) != 0 ||
```

Comment:
util_isinf introduces a insymmetry compared the the C89 HUGE_VAL constant. 

How about this:
```suggestion
            util_isnormal(averageLufs) ||
```
We can only really deal with a normal value. 
The alternative would be to check for HUGE_VAL instead. 

There is by the way the issue that util_isnormal() shall return bool, not int.

Reply:
Thanks, I've prepended a commit to fix the return types in fpclassify and use `util_isnormal()` here